modules/ROOT/pages/questions/question-19.adoc

```diff
@@ -0,0 +1,31 @@
+19. A state of gentle confusion as much as a question.
```

Comment:
@ltrouton @metricjs

Reply:
This was discovered when we tried to support using the tabfocus plugin in RTC. The plugin had long since not done much of anything; it was designed for older browsers. The entire plugin was replaced with tabindex on the editor content in TINY-8315.

This did indeed happen completely independently of the rename and deprecation efforts, those should have been cancelled when we decided to remove it.

Reply:
I already wrote Release Notes and Migration Guide copy for this. I'll try to track it down...

Reply:
This was for the old markdown-based docs but the PR is here: https://github.com/tinymce/tinymce-docs/pull/2183